kbase-extension/static/kbase/js/common/cellComponents/paramsWidget.js

```diff
@@ -87,225 +75,232 @@ define([
           field, this would be the list of all parameters meant to be output objects, so their names can
           be cross-validated for uniqueness (optional)
       */
-        function makeFieldWidget(appSpec, parameterSpec, value, closeParameters) {
-            return paramResolver.loadInputControl(parameterSpec)
```

Comment:
similar with filePaths, moved the call to paramResolver out of this method so it can be synchronous